doc/source/notebooks/advanced/mcmc.pct.py

```diff
@@ -589,3 +598,92 @@ def run_chain_fn():
     ax.hist(parameter_samples[name_to_index[hyp]], bins=20)
     ax.set_title(hyp)
 plt.tight_layout()
+
+# %% [markdown]
+# ## Prior on constraint and unconstrained parameters
+
+# %% [markdown]
+# GPflow `Parameter` class provides options for setting a prior. `Parameter` wraps a constrained tensors and provides a computation of a gradient with respect to uncosntrained transformation of that tensor. The user can set a prior either in **constrained** space or **unconstrained** space.
+
+# %% [markdown]
+# By default prior for the `Parameter` is set for _constrained_ space. To set a prior do the following:
+
+# %%
+prior_distribution = tfd.Normal(f64(0.0), f64(1.0))
+_ = gpflow.Parameter(1.0, prior_on="unconstrained", prior=prior_distribution)
+_ = gpflow.Parameter(1.0, prior_on="constrained", prior=prior_distribution)
+
+# %% [markdown]
+# When a prior is set in *constrained* space, the `gpflow.optimizers.SamplingHelper` compensates the prior density with a Jacobian of the transformation used by the parameter. For priors in the `unconstrained` space the `gpflow.optimizers.SamplingHelper` will not apply adjustments to the prior density.
+
+# %% [markdown]
+# Below we repeat the same experiment that we ran in the beginning, but with some priors in defined in the `unconstrained` space. Since we are using an Exp transform, a lognormal prior on a constrained parameter corresponds to a Normal prior on the unconstrained space:
```

Comment:
```suggestion
# Below we repeat the same experiment as before, but with some priors defined in the `unconstrained` space. Since we are using an "exp" transform, a Log-normal prior on a constrained parameter corresponds to a Normal prior on the unconstrained space:
```